test/spinxsk/spinxsk.c

```diff
@@ -2033,8 +2026,10 @@ InjectFnmpRxPacket(_In_ FNMP_HANDLE Fnmp) {
     const ETHERNET_ADDRESS localMac = FNMP_LOCAL_ETHERNET_ADDRESS_INIT;
     const ETHERNET_ADDRESS remoteMac = FNMP_NEIGHBOR_ETHERNET_ADDRESS_INIT;
 
-    const ULONG payloadBufferSize = 64'000;
+    const UINT32 payloadBufferSize = 64'000;
     UCHAR* payload = calloc(payloadBufferSize, sizeof(UCHAR));
+    payload[0] = (UCHAR)RandUlong(); // Makes TVS happy, the payload being garbage data is a plus here.
```

Comment:
Three nits: `TVS` is not a public term, `garbage` is pejorative, and we should either fully initialize the buffer or leave it as undefined, not partially initialize it to fool an easily-duped static analyzer.